### PR TITLE
add check node info on add new topic and reply

### PR DESCRIPTION
**Pull request by arthur-jx**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@arthur-jx fix:

image

**Comment by hsluoyz** [comment]
@arthur-jx what's this for?

**Comment by hsluoyz** [comment]
@kocoler @Kininaru plz review.

File: object/mailing_list.go [hunk 1]
```diff
@@ -144,15 +144,13 @@ func SyncAllNodeFromGoogleGroup() {
 	}
 }
 
-func (r Reply) AddReplyToMailingList() {
-	targetTopic := GetTopic(r.TopicId)
-	targetNode := GetNode(targetTopic.NodeId)
+func (r Reply) AddReplyToMailingList(title string, targetNode *Node) {
```
Comment (keainye) [inline object/mailing_list.go:147]: We'd better transfer `targetTopic` here. We may use some of its member variables in the future.

**Review by keainye — CHANGES_REQUESTED** [review]

File: object/mailing_list.go (hunk 1 above)
Comment (keainye) [reply object/mailing_list.go:147]: And please resolve the failed checks.

**Comment by keainye** [comment]
> @arthur-jx what's this for?

This PR is to make sure that the father topic of a new reply exists and the father node of a new topic exists.

**Comment by arthur-jx** [comment]
This is an obvious bug, why not accept my fix?

**Comment by hsluoyz** [comment]
@arthur-jx this is because the code quality of this PR is below the bar. You can send GitHub issue and we will assign community members to make PR.

**Comment by arthur-jx** [comment]
@hsluoyz  Since the code quality of current community members is so high, why are there so many low-level bugs?

**Comment by hsluoyz** [comment]
@arthur-jx which ones? you can friendly point out the low-level bugs, we will fix them one by one.

**Pull request closed without merging**